### PR TITLE
Updated broken links of Table of contents

### LINKED ISSUE #125: Broken Table of Contents links

### PR DESCRIPTION
**Pull request by dhirajchaurasiya10**
Fixes #125

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +25,9 @@
 
 ## 📚 Table of Contents
 
-- [📖 Books](#books)
-- [📅 Calendars](#calendars)
-- [🃏 Cards](#cards)
+- [📖 Books](#📖-books)
+- [📅 Calendars](#📅-calendars)
+- [🃏 Cards](#🃏-cards)
```
Comment (jbampton) [inline README.md:30]:
```suggestion
- [🃏 Cards](#--cards)
```

**Review by jbampton — CHANGES_REQUESTED** [review]
The links still don't work.

I tried an edit just now too

**Comment by jbampton** [comment]
Please only one feature per PR

**Comment by dhirajchaurasiya10** [comment]
sorry I messed it by mistake

**Comment by dhirajchaurasiya10** [comment]
Can you revert the last 2 commits. If not close this and let me open a separate PR for this issue

**Pull request closed without merging**